R/function-filtering.R

```diff
@@ -0,0 +1,142 @@
+#' @title Basic filtering functions for metabolomics
```

Comment:
hm yes, now I see get your thoughts on the naming of the file. Indeed, filtering does not make much sense, maybe use "quality-assessment.R" for the file name and here `Basic quality assessment functions for metabolomics" as title?